models/login_source.go

```diff
@@ -403,6 +403,20 @@ func LoginViaLDAP(user *User, login, password string, source *LoginSource, autoR
 
 	var isAttributeSSHPublicKeySet = len(strings.TrimSpace(source.LDAP().AttributeSSHPublicKey)) > 0
 
+	// Update User if exist
+	isExist, err := IsUserExist(0, sr.Username)
+	if err != nil {
+		return nil, err
+	} else if isExist {
+		user.FullName = composeFullName(sr.Name, sr.Surname, sr.Username)
+		user.Email = sr.Mail
+		// Change existing admin flag only if AdminFilter option is set
+		if len(source.LDAP().AdminFilter) > 0 {
```

Comment:
Should a LDAP administrator automatically be an admin of Gitea?

Reply:
> 
> 
> Should a LDAP administrator automatically be an admin of Gitea?

https://docs.gitea.io/en-us/authentication/ 😉 
> Admin Filter (optional)
>    An LDAP filter specifying if a user should be given administrator privileges. If a user account passes the filter, the user will be privileged as an administrator.

Reply:
OKay! Missed that document.